docs/faq.md

```diff
@@ -250,3 +250,14 @@ to:
 ```
 
 </details>
+
+## Scully Command Line Interface
+
+<details>
+<summary>Why are the pair of hyphens - `--` - used when running `yarn scully`?</summary>
```

Comment:
yarn doesn't have this issue, its strictly NPM.

Reply:
Ah, I see. Will make changes accordingly.